src/plugins/intel_gpu/src/plugin/ops/adaptive_pooling.cpp

```diff
@@ -63,7 +63,6 @@ static void CreateAdaptiveMaxPoolOp(ProgramBuilder& p, const std::shared_ptr<ov:
                                          inputs[0],
                                          inputs[1],
                                          cldnn::element_type_to_data_type(op->get_index_element_type()),
-                                         cldnn::padding({0, 0, 0, 0}, 0),
                                          cldnn::element_type_to_data_type(op->get_output_element_type(0)),
                                          num_outputs};
         poolPrim.output_paddings = get_output_paddings();
```

Comment:
`poolPrim.output_paddings = get_output_paddings();`
Maybe this is also not needed since it is not used 
The same in many other primitives: ctc_greedy_decoder, proposal, topk, nms, experimental_detectron ops

Reply:
Removed